config/application.rb

```diff
@@ -37,12 +37,7 @@ class Application < Rails::Application
 
     config.active_job.queue_adapter = :sidekiq
 
-    config.lograge.enabled = true
-    config.lograge.enabled = Lograge::Formatters::Json.new
-    config.log_formatter = proc do |severity, datetime, progname, msg|
-      "time=\"#{datetime.iso8601(3)}\" level=\"#{severity}\" pid=\"#{Process.pid}\" #{msg.to_s.gsub('"', "'")}\n"
-    end
-
```

Comment:
is this a side-effect of some mistakenly committed file ? or its intentional ? @namrataukirde

Reply:
Let me check @abhinav098. Thanks for mentioning it. :+1:

Reply:
Yes, mistakenly deleted I think. We need to re-add it. Thanks :)

Reply:
@steveyken, We removed it intentionally or we need it? Please guide us on this.

Reply:
```
    config.lograge.enabled = true
    config.lograge.enabled = Lograge::Formatters::Json.new
```

this looks wrong to begin with

Reply:
yeah well spotted patrick. it was definitely wrong code to start with but I moved everything to an initializer. see log_formatting.rb

Reply:
and thanks for picking it up @abhinav098